### PR TITLE
Feature/add decidable identity attribute value renderer

### PR DESCRIPTION
**Pull request by aince42**

### CODE REVIEW

File: packages/renderers/example/lib/pages/home_screen.dart [hunk 1]
```diff
@@ -85,7 +86,14 @@ class _HomeScreenState extends State<HomeScreen> {
           title: 'Rejected Create Attribute Request JSON Example',
           pageBuilder: (context) => const RejectedCreateAttributeRequestJsonExample(),
         ),
-      ])
+      ]),
+      _MenuGroup(title: 'Dedicdable Read Attriubute', items: [
```
Comment (Vin-RM) [inline packages/renderers/example/lib/pages/home_screen.dart:90]: Typo:
```suggestion
      _MenuGroup(title: 'Decidable Read Attribute', items: [
```

**Review by Vin-RM — CHANGES_REQUESTED** [review]

**Review by Vin-RM — APPROVED** [review]

File: packages/renderers/lib/src/renderers/request_item_renderer/decidable/widgets/identity_attribute_value_renderer.dart [hunk 2]
```diff
@@ -16,124 +16,158 @@ class IdentityAttributeValueRenderer extends StatelessWidget {
...
-        const Text('Affiliation'),
-        Text(value.role),
-        Text(value.organization),
-        Text(value.unit),
+    return ComplexAttributeListTile(
+      title: 'i18n://attributes.values.Affiliation._title',
+      titles: const [
+        'i18n://attributes.values.Affiliation.role.label',
+        'i18n://attributes.values.Affiliation.organization.label',
+        'i18n://attributes.values.Affiliation.unit.label',
```
[the remark this answers is not in the document]
Comment (jkoenig134) [reply packages/renderers/lib/src/renderers/request_item_renderer/decidable/widgets/identity_attribute_value_renderer.dart:68]: this counts for every i18n string in the file

**Review by jkoenig134 — CHANGES_REQUESTED** [review]
subtitle is a real english word. please use `subtitle` instead of `subTitle` everywhere.

File: packages/renderers/lib/src/renderers/widgets/complex_attribute_list_tile.dart [hunk 3]
```diff
@@ -31,14 +42,19 @@ class ComplexAttributeListTile extends StatelessWidget {
...
                   return Column(
                     crossAxisAlignment: CrossAxisAlignment.start,
                     children: [
-                      TranslatedText('${titles[index]}:', style: const TextStyle(fontSize: 12, color: Color(0xFF42474E))),
+                      Row(
+                        children: [
+                          TranslatedText('${listOfTitles[index]}', style: titlesTextStyle),
+                          const Text(':', style: titlesTextStyle),
+                        ],
+                      ),
```
[the remark this answers is not in the document]
Comment (aince42) [reply packages/renderers/lib/src/renderers/widgets/complex_attribute_list_tile.dart:55]: I tried it, but unfortunately it is not possible.
We could translate the text outside of the widget, but I think this is the better way.

[the remark this answers is not in the document]
Comment (jkoenig134) [reply packages/renderers/lib/src/renderers/widgets/complex_attribute_list_tile.dart:55]: nope, translate outside is totally fine!

File: packages/renderers/lib/src/renderers/widgets/complex_attribute_list_tile.dart [hunk 4]
```diff
@@ -4,14 +4,25 @@ import 'package:translated_text/translated_text.dart';
...
+    const titlesTextStyle = TextStyle(fontSize: 12, color: Color(0xFF42474E));
+
+    final listOfTitles = [];
+    final listOfSubTitles = [];
+    for (int i = 0; i < subTitles.length; i++) {
+      if (subTitles[i] != null) {
+        listOfTitles.add(titles[i]);
+        listOfSubTitles.add(subTitles[i]!);
+      }
+    }
```
[the remark this answers is not in the document]
Comment (jkoenig134) [reply packages/renderers/lib/src/renderers/widgets/complex_attribute_list_tile.dart:24]: this messes up the order of subtitles and titles, I really don't get what you are doing here.

File: packages/renderers/lib/src/renderers/widgets/custom_list_tile.dart
Comment (jkoenig134) [inline packages/renderers/lib/src/renderers/widgets/custom_list_tile.dart:1]: look at nicoles PR for naming. you should talk about that or you will have clashes when merging.

File: packages/renderers/lib/src/renderers/request_item_renderer/widgets/query_renderer.dart [hunk 5]
```diff
@@ -10,7 +10,7 @@ class IdentityAttributeQueryRenderer extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return CustomListTile(title: '', subTitle: query.name);
+    return CustomListTile(title: '', value: query.name);
```
Comment (jkoenig134) [inline packages/renderers/lib/src/renderers/request_item_renderer/widgets/query_renderer.dart:13]: I am confused if this empty titles in this file.

File: packages/renderers/lib/src/renderers/request_item_renderer/widgets/relationship_attribute_renderer.dart [hunk 6]
```diff
@@ -44,7 +44,7 @@ class _ConsentAttributeValueRenderer extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return CustomListTile(title: '', subTitle: value.consent);
+    return CustomListTile(title: '', value: value.consent);
```
Comment (jkoenig134) [inline packages/renderers/lib/src/renderers/request_item_renderer/widgets/relationship_attribute_renderer.dart:47]: empty titles in this file

File: packages/renderers/lib/src/renderers/widgets/custom_list_tile.dart [hunk 7]
```diff
@@ -23,7 +23,7 @@ class CustomListTile extends StatelessWidget {
                   children: [
                     TranslatedText(title, style: const TextStyle(fontSize: 12, color: Color(0xFF42474E))),
                     const SizedBox(height: 2),
-                    Text(subTitle, style: const TextStyle(fontSize: 16)),
+                    TranslatedText(value, style: const TextStyle(fontSize: 16)),
```
Comment (jkoenig134) [inline packages/renderers/lib/src/renderers/widgets/custom_list_tile.dart:26]: why translating the value?

**Review by jkoenig134 — APPROVED** [review]
I Approve that this can finally be merged - but please look at the not-outdated comments and resolve them in @nicole-eb's PR before you assign me for review there.

Also my comment about subTitle -> subtitle is still open, please resolve this (if you don't remove that).

**Pull request merged**